modules/ding_user_frontend/ding_user_frontend.module

```diff
@@ -296,6 +291,9 @@ function ding_user_frontend_url_outbound_alter(&$path, &$options, $original_path
  * ding_user_frontend_build_uniform_menu_link().
  */
 function ding_user_frontend_url_inbound_alter(&$path, $original_path, $path_language) {
+  // Store the path. This allows us to check whether we modified it or not.
+  ding_user_frontend_original_path($original_path);
```

Comment:
@kasperg why not use request_path() to get the original URL?

Reply:
It looks like the `$path` variable passed to this function, might not even be the path the the user requested, since it will check to see if it's an aliased normal path before invoking hook: https://api.drupal.org/api/drupal/includes%21path.inc/function/drupal_get_normal_path/7.x

Reply:
I hear you concerns, but it have been decided that this should be sent to test now. So please write this concerns into the issue tracker.

Reply:
Nevermind :) This will work anyway. I think using `$request_path` would just have made it a bit cleaner, but if it needs to be included right now this will do.

And assuming the `$path` variable is original path in inbound_alter() will only be a "problem" if a provider user path is aliased and the only effect will be a redirect to a user/me path, which is unneeded, since the alias prob wont contain an ID.